ci/benchmark_bencher_baseline.yml

```diff
@@ -0,0 +1,62 @@
+include:
+  - local: 'ci/base.yml'
+
+.benchmark_model_stencils:
+  stage: benchmark
+  script:
+    - | # Run the benchmarks
+      bencher run \
+      --project "$BENCHER_PROJECT" \
+      --token "$BENCHER_API_TOKEN" \
+      --branch main \
+      --testbed "ci-runner:${SYSTEM_NAME}:${BACKEND}:${GRID}" \
+      --threshold-measure latency \
+      --threshold-test percentage \
+      --threshold-max-sample-size 64 \
+      --threshold-upper-boundary 0.1 \
+      --thresholds-reset \
+      --err \
+      --adapter python_pytest \
+      --file results.json \
+      "nox -s benchmark_model-3.10 -- --backend=$BACKEND --grid=$GRID"
+    - | # Script to flatten the results.json file. Needed for Kibana.
```

Comment:
I think it's essential to extract the kibana related setup into a script for maintainability. But, actually this could be even removed in this first setup.

Reply:
I thought exactly the same. Possibly, we will skip the Kibana service and keep only the self-hosting stuff. So for now, I would leave it as is.

Reply:
No, I meant skip this setup. Before merge we should make it clean or remove it.

Reply:
I skip the kibana setup for now, and we can add it if needed.